pyoozie/oozie_api.py

```diff
@@ -0,0 +1,561 @@
+from __future__ import unicode_literals
+
+import logging
+import requests
+import simplejson as json
+
+from pyoozie.exceptions import OozieException
+from pyoozie.model import ArtifactType, Coordinator, CoordinatorAction, \
+    Workflow, WorkflowAction, parse_coordinator_id, parse_workflow_id
+from pyoozie.builder import _coordinator_submission_xml, _workflow_submission_xml
+
+logger = logging.getLogger('pyoozie.OozieAPI')
+logging.getLogger('requests').setLevel(logging.WARNING)
+
+
+class OozieAPI(object):
```

Comment:
Should we rename this `Api`? Or some other noun, e.g. `Session` as in `requests`? Or perhaps `Client` and wrap it in a factory called `client`?  The `Oozie` might be redundant, e.g.:

```
oozie = pyoozie.OozieAPI(url='https://...', user='oozie', timeout=10)
```

As opposed to:


```
oozie_client = pyoozie.client(url='https://...', user='oozie', timeout=10)
```